.github/workflows/serverless-integration.yml

```diff
@@ -0,0 +1,39 @@
+name: "Serverless Integration Tests"
+
+on:
+  push:
+    paths:
+      - 'cmd/serverless/**'
+      - 'pkg/serverless/**'
+      - 'test/integration/serverless/**'
```

Comment:
I would add the current file `.github/workflows/serverless-integration.yml` as well? As if this changes, we might want to run integration tests as well?